pilot/pkg/security/authz/model/generator.go

```diff
@@ -45,8 +44,7 @@ func (destIPGenerator) principal(_, _ string, _ bool) (*rbacpb.Principal, error)
 	return nil, fmt.Errorf("unimplemented")
 }
 
-type destPortGenerator struct {
-}
+type destPortGenerator struct{}
```

Comment:
prefer to not change all these unrelated files

Reply:
Thanks! Remove all unrelated changes introduced by local format on `make gen`,